recipes/poolsnp/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set version = "1.0.0" %}
+
+package: 
+  name: poolsnp
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/capoony/PoolSNP.git
+  sha256: c630eb7d44805636322839e245fb7f566af737513a3170396b62a50018aa0008
+
+requirements:
+  host:
+    - python >=3
+    - parallel
+  run:
+    - python >=3
```

```suggestion
    - python
```
Py2k is no more supported anyway. At the moment Bioconda builds with 3.8. 3.9, 3.10, 3.11 and 3.12.

---

Made the necessary modifications